src/map/clif.c

```diff
@@ -1155,15 +1155,15 @@ static void clif_set_unit_idle(struct block_list *bl, struct map_session_data *t
 	p.font = (sd) ? sd->status.font : 0;
 #endif
 #if PACKETVER >= 20120221
-	if (battle_config.show_monster_hp_bar && bl->type == BL_MOB && status_get_hp(bl) < status_get_max_hp(bl)) {
+	const struct mob_data *md = BL_UCCAST(BL_MOB, bl);
```

Comment:
here check for mob was removed.
BL_UCCAST not check is bl mob or not, it cast any ``bl`` to ``mob_data``

Reply:
Please see changes in last commit